lib/stream-chat/indifferent_hash.rb

```diff
@@ -0,0 +1,192 @@
+# Source from https://github.com/sinatra/sinatra/blob/master/lib/sinatra/indifferent_hash.rb
```

Comment:
For a copy of this size we should be maintaining license file. Can this go in a subfolder along with the LICENSE from the original project?

Reply:
Is it not enough to provide link? the code have MIT license

Reply:
No, we have to distribute a copy of the license and the copyright info with the code. The alternative is to copy the license file that is [currently distributed with the project](https://github.com/sinatra/sinatra/blob/master/LICENSE) and put it in a comment at the top of this file.